src/ResponseFactory.php

```diff
@@ -9,7 +9,6 @@ class ResponseFactory implements ResponseFactoryInterface
 {
     public function createResponse($code = 200)
     {
-        return (new Response())
-            ->withStatus($code);
+        return new Response('php://memory', $code);
```

Comment:
And what if the default changes to something undesirable in the future? Having one less object isn't worth it in my opinion.